### ISSUE TITLE
Name is not defined for "Change Equation Style" pop-up when activating it.

### ISSUE THREAD

**Issue by MicrosoftIssueBot** [microsoft/calculator#1674]
Repro steps:
Open the Application Calculator
Navigate to ‘Graphing’ button and activate it.
Now navigate to the any Equation and press "Shift+F10" key 
Now navigate to the 'Change Equation Style' and activate it.
Observe the issue.

Actual Result:
Name is not defined for "Change Equation Style" pop-up when activating it. Screen reader announce "Window pop-up"

Expected Result:
Name should be defined for "Change Equation Style" pop-up when activating it. Screen reader should announce "Change Equation Style pop-up".
